### ISSUE TITLE
Installation Could Not Be Resolved

### ISSUE THREAD

**Issue by sorensenj50** [Neo4j-Swift/Neo4j-Swift#11]
I got a "Fail to Resolve Dependency Error" after trying to install package. See [this](https://stackoverflow.com/questions/69790488/swift-package-could-not-be-resolved) stack overflow post. 

P.S. You said in the README that stack overflow and the slack channel were the preferred mediums to communicate bugs. Note that I posted in stack overflow (it didn't allow me to use the Theo tag--I didn't have enough rep), and the link to the slack channel lead me to a general neo4j discord. I could not find a channel for Theo specifically in that discord server. Just letting you know that I tried other places first...

**Comment by joshjacob** [comment]
@sorensenj50 I just did a quick Xcode iOS project from scratch to look at another issue and used the Xcode package tools. Were you using the Xcode UI for adding the package?

My steps:
1. File > App Packages...
2. In the package window, paste the GitHub URL in to search field in top right: `https://github.com/Neo4j-Swift/Neo4j-Swift`
3. In the search result for Neo4j-Swift, select "Branch" as Dependency Rule and enter `dev/5.2.1` for latest branch
4. Hit "Add Package" button in lower right
5. In "Choose Package Products..." modal, select the target and hit "Add Package" again.

In the "Report Navigator" of Xcode where you usually see build logs, they will list package manager results under the project. Do you see one with a red error X on it?

**Comment by sorensenj50** [comment]
Yes, I used the UI / Swift Package Manager. I think I deleted the project where I tried to install it so can't verify anything anymore.

**Comment by niklassaers** [comment]
Closing this old issue. With Theo 6.0 released, please use Swift Package Manager with the stable release:

```swift
.package(url: "https://github.com/Neo4j-Swift/Neo4j-Swift.git", from: "6.0.0")
```

If you encounter installation issues with 6.0, please open a new issue.
